src/benchmarklib/benchmark_runner.cpp

```diff
@@ -154,7 +154,7 @@ void BenchmarkRunner::run() {
     }
   }
 
-  _table_generator->delete_binaries();
+  //_table_generator->delete_binaries();
```

Comment:
This is probably commented out so that you can reuse the binaries right? Some nicer structure - with an extra directory for the persisted data files (binary) would be great.